[2024-12-24T00:27:30.647+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-24T00:27:30.695+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: NY_Pipeline_ETL_and_ML_Training.step_create_original_table manual__2024-12-24T00:27:28.773677+00:00 [queued]>
[2024-12-24T00:27:30.717+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: NY_Pipeline_ETL_and_ML_Training.step_create_original_table manual__2024-12-24T00:27:28.773677+00:00 [queued]>
[2024-12-24T00:27:30.721+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-12-24T00:27:30.751+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): step_create_original_table> on 2024-12-24 00:27:28.773677+00:00
[2024-12-24T00:27:30.823+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'NY_Pipeline_ETL_and_ML_Training', 'step_create_original_table', 'manual__2024-12-24T00:27:28.773677+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/etl_train_pipeline.py', '--cfg-path', '/tmp/tmpka66rymb']
[2024-12-24T00:27:30.839+0000] {standard_task_runner.py:105} INFO - Job 36: Subtask step_create_original_table
[2024-12-24T00:27:30.853+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=947) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-24T00:27:30.854+0000] {standard_task_runner.py:72} INFO - Started process 949 to run task
[2024-12-24T00:27:30.966+0000] {task_command.py:467} INFO - Running <TaskInstance: NY_Pipeline_ETL_and_ML_Training.step_create_original_table manual__2024-12-24T00:27:28.773677+00:00 [running]> on host 8a0d5a4738c0
[2024-12-24T00:27:31.149+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Artur Tiburski' AIRFLOW_CTX_DAG_ID='NY_Pipeline_ETL_and_ML_Training' AIRFLOW_CTX_TASK_ID='step_create_original_table' AIRFLOW_CTX_EXECUTION_DATE='2024-12-24T00:27:28.773677+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-24T00:27:28.773677+00:00'
[2024-12-24T00:27:31.151+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-24T00:27:33.648+0000] {logging_mixin.py:190} WARNING - /opt/***/***_server/dags/etl_train_pipeline_modules/dsa_carrega_dados.py:55 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2024-12-24T00:27:33.649+0000] {logging_mixin.py:190} INFO - An Error occurred: BROKERTITLE (VARCHAR(255)) not a string
[2024-12-24T00:27:33.654+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-24T00:27:33.669+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-24T00:27:33.671+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=NY_Pipeline_ETL_and_ML_Training, task_id=step_create_original_table, run_id=manual__2024-12-24T00:27:28.773677+00:00, execution_date=20241224T002728, start_date=20241224T002730, end_date=20241224T002733
[2024-12-24T00:27:33.727+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-24T00:27:33.752+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-24T00:27:33.756+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
